rubygem-smart_proxy_dns_infoblox/rubygem-smart_proxy_dns_infoblox.spec

```diff
@@ -0,0 +1,96 @@
+# Generated from smart_proxy_dns_infoblox-0.0.3.gem by gem2rpm -*- rpm-spec -*-
+%global gem_name smart_proxy_dns_infoblox
+%global plugin_name dns_infoblox
+
+%global foreman_proxy_dir %{_datarootdir}/foreman-proxy
+%global foreman_proxy_bundlerd_dir %{foreman_proxy_dir}/bundler.d
+%global foreman_proxy_settingsd_dir %{_sysconfdir}/foreman-proxy/settings.d
+
+Name: rubygem-%{gem_name}
+Version: 0.0.4
+Release: 1%{?foremandist}%{?dist}
+Summary: Infoblox DNS provider plugin for Foreman's smart proxy
+Group: Applications/Internet
+License: GPLv3
+URL: https://github.com/sjoeboo/smart_proxy_dns_infoblox
+Source0: http://rubygems.org/gems/%{gem_name}-%{version}.gem
+Requires: foreman-proxy >= 1.13.0
+%if 0%{?rhel} == 6
+Requires: ruby(abi)
+%else
+Requires: ruby(release)
+%endif
+Requires: ruby
+Requires: ruby(rubygems)
+Requires: rubygem(infoblox)
```

Comment:
This isn't listed in the runtime deps on the gem, please correct it for future releases.